### PR TITLE
BMCP-3: Adding Dependencies

### PR DESCRIPTION
**Pull request by nikhilsharma0025**
## Description
 - Added different dependencies that we are using
 
#### Modification
- package.json

### CODE REVIEW

**Review by 2SkeePeR2 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,14 +3,20 @@
...
   "dependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^11.2.7",
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.58",
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
```
Comment (nik-1236789) [inline package.json:12]: react-router-dom??

Comment (nik-1236789) [inline package.json:12]: react-router-dom???

**Review by nik-1236789 — APPROVED** [review]
Add react-router-dom in dependencies and it's good to go.

**Review by Aditya413 — APPROVED** [review]

**Comment by 2SkeePeR2** [comment]
@nik-1236789 consider approving PR after required packages have been added,

**Review by nik-1236789 — APPROVED** [review]

**Pull request merged**